### DISCUSSION TITLE
[Request] Plugin or integration of ct

### DISCUSSION THREAD

**Discussion by eugenov** [Eugeny/tabby#6571] [General]
Hello!
As many asked to color output syntax, there is already a pypi plugin that does exactly what they say: https://github.com/hSaria/ChromaTerm
Is there any way to integrate it? I know dev's position about highlighting - "it is shell's job", but as user of hundreds of hosts with various oses I would say "it's terminal's job" :) Could someone experienced look at this problem?
Thank you dev for your hard work!

**Comment by Eugeny** [comment]
Using CT for local sessions should be easy enough (e.g. replace `zsh` with `ct zsh` in a profile's settings), but it won't work for SSH because Tabby isn't an OpenSSH wrapper.

**Reply by eugenov** [reply]
It would be ok to use ct with local PS shell if conpty deals with alternate screen, but it does not...

**Comment by 0n11san** [comment]
<img width="898" alt="Screen Shot 2022-10-11 at 12 40 52 AM" src="https://user-images.githubusercontent.com/9022899/194998908-d45083fd-b5f5-4b2f-b0ce-ca408f34b27e.png">


I'm guessing my result is not quite what you folks had in mind?

It would be nice to have this pre-saved in a profile, but I suppose w/ this workaround (where I manually log in each time in order to be able to use 'ct') ultimately gets me what I'm looking for... albeit w/ some quality of life improvements missing (like no SFTP --- see below image where i'm using Tabby's profiles instead)

<img width="956" alt="Screen Shot 2022-10-11 at 12 47 59 AM" src="https://user-images.githubusercontent.com/9022899/194999645-de227768-c60d-4783-a9ab-cf1a896e9a54.png">
